src/components/AnnotationPopover/AnnotationPopover.scss

```diff
@@ -1,8 +1,43 @@
 @import '../../variables';
 
 .ba {
-    .ba-annotation-popover {
+    .ba-popover {
         cursor: auto; // Overrides cursor: none from annotation mode
+        max-width: 282px;
+        position: absolute;
+        z-index: 3;
+    }
+
+    .ba-popover-caret {
+        left: 50%;
+        position: relative;
+        z-index: 3;
+
+        // CSS arrow for caret above container
```

Comment:
You might be able to use the BRUI dropdown instead of manually adding this caret.

Reply:
I'll keep a separate task to later look into using the BRUI Flyout component with either the point annotation icon, drawing boundary, or an invisible highlight anchor element (I would need to add that in)